src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -526,13 +526,20 @@ export const EVENT_TYPES = {
   hearingHeld: 'hearing_held',
   hearingCancelled: 'hearing_cancelled',
   hearingNoShow: 'hearing_no_show',
+  transcript: 'transcript',
   bvaDecision: 'bva_decision',
   bvaRemand: 'bva_remand',
   withdrawn: 'withdrawn',
   merged: 'merged',
   cavcDecision: 'cavc_decision',
   recordDesignation: 'record_designation',
-  reconsideration: 'reconsideration'
+  reconsideration: 'reconsideration',
+  rampNotice: 'ramp_notice',
+  rampOptIn: 'ramp',
+  otherClose: 'other_close',
+  vacated: 'vacated',
+  death: 'death',
+  failureToRespond: 'ftr'
```

Comment:
@gnakm there are some new events in the design file, should they be added here?

Reply:
@cvalarida how do we feel about these variable names? Should rampOptIn and failureToRespond be shortened so that they match their values? Or is it helpful to have more descriptive names?

Reply:
I like the expanded names like you've got it. :+1:
`Clarity from verbosity > confusion from brevity` in my opinion.